src/main/java/com/google/firebase/messaging/AndroidNotification.java

```diff
@@ -180,6 +182,15 @@ private AndroidNotification(Builder builder) {
     SECRET,
   }
 
+  private static final Map<Priority, String> PRIORITY_MAP = 
```

Comment:
Nit: Move to the top of the class definition

Reply:
Done